sources/hubspot/__init__.py

```diff
@@ -38,54 +38,181 @@
+    DEFAULT_QUOTE_PROPS,
     OBJECT_TYPE_PLURAL,
+    STARTDATE,
+    WEB_ANALYTICS_EVENTS_ENDPOINT,
 )
 
 THubspotObjectType = Literal["company", "contact", "deal", "ticket", "product", "quote"]
 
 
 @dlt.source(name="hubspot")
 def hubspot(
-    api_key: str = dlt.secrets.value, include_history: bool = False
+    api_key: str = dlt.secrets.value,
+    include_history: bool = False,
 ) -> Sequence[DltResource]:
     """
-    A DLT source that retrieves data from the HubSpot API using the specified API key.
+    A DLT source that retrieves data from the HubSpot API using the
+    specified API key.
 
-    This function retrieves data for several HubSpot API endpoints, including companies, contacts, deals,
-    tickets, products and web analytics events. It returns a tuple of Dlt resources, one for each endpoint.
+    This function retrieves data for several HubSpot API endpoints,
+    including companies, contacts, deals, tickets, products and web
+    analytics events. It returns a tuple of Dlt resources, one for
+    each endpoint.
 
     Args:
-        api_key (str, optional): The API key used to authenticate with the HubSpot API. Defaults to dlt.secrets.value.
-        include_history: Whether to load history of property changes along with entities. The history entries are loaded to separate tables.
+        api_key (Optional[str]):
+            The API key used to authenticate with the HubSpot API. Defaults
+            to dlt.secrets.value.
+        include_history (Optional[bool]):
+            Whether to load history of property changes along with entities.
+            The history entries are loaded to separate tables.
 
     Returns:
-        tuple: A tuple of Dlt resources, one for each HubSpot API endpoint.
+        Sequence[DltResource]: Dlt resources, one for each HubSpot API endpoint.
 
     Notes:
-        This function uses the `fetch_data` function to retrieve data from the HubSpot CRM API. The API key
-        is passed to `fetch_data` as the `api_key` argument.
+        This function uses the `fetch_data` function to retrieve data from the
+        HubSpot CRM API. The API key is passed to `fetch_data` as the
+        `api_key` argument.
     """
-    return [
-        companies(include_history=include_history),
-        contacts(include_history=include_history),
-        deals(include_history=include_history),
-        tickets(include_history=include_history),
-        products(include_history=include_history),
-        quotes(include_history=include_history),
-    ]
+
+    @dlt.resource(name="companies", write_disposition="replace")
+    def companies(
+        api_key: str = api_key,
+        include_history: bool = include_history,
+        props: Sequence[str] = DEFAULT_COMPANY_PROPS,
+        include_custom_props: bool = True,
+    ) -> Iterator[TDataItems]:
+        """Hubspot companies resource"""
+        yield from crm_objects(
+            "company",
+            api_key,
+            include_history=False,
```

Comment:
include_history always is False for companies?

Reply:
It was done without any explanations here: https://github.com/dlt-hub/verified-sources/pull/206
Looks like a testing change, which was not reverted before committing.